contents/stats_distributions.md

```diff
@@ -0,0 +1,62 @@
+## Distributions {#sec:stats_dist}
+
+Before defining distributions, we must talk about **probability**.
+
+**The probability of an event is a real number ($\in \mathbb{R}$) between 0 and 1, where, roughly, 0 indicates the impossibility of the event and 1 indicates the certainty of the event**.
+The greater the likelihood of an event, the more likely it is that the event will occur.
+A simple example is the tossing of a fair (impartial) coin.
+Since the coin is fair, both results ("heads" and "tails") are equally likely; the probability of "heads" is equal to the probability of "tails"; and since no other result is possible, the probability of "heads" or "tails" is $\frac{1}{2}$ (which can also be written as 0.5 or 50%).
+
+Regarding notation, **we define $A$ as an event and $P(A)$ as the probability of event $A$**, thus:
+
+$$ \{P(A) \in \mathbb{R} : 0 \leq P(A) \leq 1 \}. $$ {#eq:probability}
+
+This means the "probability of the event to occur is the set of all real numbers between 0 and 1; including 0 and 1".
```

Comment:
is the set of all real numbers in the close interval $[0,1]$. (but probably too formal).

Reply:
We can stick with this definition because it plays nice with the Kolmogorov axioms below. But, yes, it is the set of all real number between 0 and 1 (both inclusive).

I will add a more intuitive explanation and also some sugarcoating stuff to explain better.